doc/subsystems/networking/net-app-api.rst

```diff
@@ -5,17 +5,23 @@ Network Application API
 
 The Network Application (net-app) API allows applications to:
 
-* **Initialize** the application for networking use. This means, for example,
+**Initialize**
+
+  The application for networking use. This means, for example,
   that if the application needs to have an IPv4 address, and if DHCPv4 is
   enabled, then the net-app API will make sure that the device will get an
   IPv4 address before the application is started.
 
-* **Set** various options for the networking subsystem. This means that if the
+**Set**
+
```

Comment:
all these blank lines need to be deleted between the term line and the definition body...